src/Tests/CppComponent/Class1.cpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
Probably want to rename CppComponent to like CsWinMDComponent so it is clear this is a CsWinMD test or moving it under a CsWinMD folder.